src/components/shared/MarkerTooltipContents.js

```diff
@@ -122,6 +122,29 @@ function _makePhaseTimesArray(
   return array;
 }
 
+function _dataPriorityReplace(priority: number): ?string {
+  var prioLabel;
+
+  // https://searchfox.org/mozilla-central/source/xpcom/threads/nsISupportsPriority.idl#24-28
+  if(priority < -10) {
+    prioLabel = 'Highest';
+  } else if ((priority > -11) && (priority < 0)){
```

Comment:
nit: I'd use the signs `<=` and `>` so that we have the same edges.

(I've read recently a nice and old write-up by, I think, Donald Knuth, about that, but can't find the link anymore)